pkg/service/service.go

```diff
@@ -397,3 +399,21 @@ func isRequired(required []string, name string) bool {
 	}
 	return false
 }
+
+// MachineReadableSuccessOutput prints out machine readable output
+func MachineReadableSuccessOutput(services []occlient.Service) {
```

Comment:
Besides the same point that I mentioned in https://github.com/openshift/odo/pull/2066#pullrequestreview-283387331, it looks like `MachineReadableSuccessOutput` in both the PRs has more code common between them than uncommon. Is it possible to have single function for these?

Reply:
With the way that yaml unmarshal's, I don't think so :(